Glyssen/Dialogs/VoiceActorInformationDlg.cs

```diff
@@ -16,5 +18,69 @@ public VoiceActorInformationDlg()
 		{
 			InitializeComponent();
 		}
+
+		private void RemoveSelectedRows(bool confirmWithUser)
+		{
+			bool deleteConfirmed = !confirmWithUser;
+
+			if (confirmWithUser)
+			{
+				deleteConfirmed = MessageBox.Show("Are you sure you want to delete these rows?", "Confirm", MessageBoxButtons.YesNo) == System.Windows.Forms.DialogResult.Yes;
+			}
+
+			if (deleteConfirmed)
+			{
+				for (int i = m_dataGrid.SelectedRows.Count - 1; i >= 0; i--)
+				{
+					m_dataGrid.Rows.Remove(m_dataGrid.SelectedRows[i]);
+				}
+
+				if(m_dataGrid.RowCount <= 1)
```

Comment:
Space after if.  You can probably change your settings in VS if it isn't doing this by default.